README.md

```diff
@@ -1,18 +1,58 @@
 # On Prem
 
-## Installing microk8s
+## Installing microk8s and tensorleap from the repo
 
-```
+### On Mac
+
+```bash
+# install microk8s
 brew install ubuntu/microk8s/microk8s
 microk8s install
+microk8s status --wait-ready
 microk8s enable ingress dns storage rbac
+
+# Update /etc/hosts
+echo "$(multipass list --format json | jq -r '.list[] | select(.name == "microk8s-vm") | .ipv4 | first')\ttensorleap.local" | sudo tee -a /etc/hosts
+
+# Add microk8s context to your local KubeConfig file
+KUBECONFIG=<(microk8s config):$HOME/.kube/config kubectl config view --raw > kubeConfig && mv kubeConfig $HOME/.kube/config
```

Comment:
```suggestion
echo "$(KUBECONFIG=<(microk8s config):$HOME/.kube/config kubectl config view --raw)" | tee $HOME/.kube/config
```
Simpler? (but needs testing)

Reply:
could break for some reason, tried that (but without `tee`)...
maybe `tee` works but we need to check (if you decided to check, back your original `KubeConfig`)